docs/master/performance/caching.md

```diff
@@ -101,3 +101,68 @@ type GithubProfile {
   repos: [Repository] @cache
 }
 ```
+
+## HTTP Cache-Control header
+
+You can change the [`Cache-Control` header](https://developer.mozilla.org/de/docs/Web/HTTP/Headers/Cache-Control) of your response
+regardless of [@cache](../api-reference/directives.md#cache)
+by adding the [@cacheControl](../api-reference/directives.md#cachecontrol) directive to a field.
+
+The final header settings are calculated based on these rules:
+
+- `max-age` equals the lowest `maxAge` among all fields. If that value is 0, `no-cache` is used instead
+- visibility is `public` unless the scope of a queried field is `PRIVATE`
+
+For more details check [Apollo](https://www.apollographql.com/docs/apollo-server/performance/caching/#calculating-cache-behavior).
+
+Given the following example schema:
+
+```graphql
+type User {
+  tasks: [Task!]! @hasMany @cacheControl(maxAge: 50, scope: PUBLIC)
+}
+
+type Company {
+  users: [User!]! @hasMany @cacheControl(maxAge: 25, scope: PUBLIC)
+}
+
+type Task {
+  id: ID @cacheControl(maxAge: 10, scope: PUBLIC)
+  description: String @cacheControl
+}
+
+type Query {
+  me: User! @auth @cacheControl(maxAge: 5, scope: PRIVATE)
+  company: Company!
+}
+```
+
+The Cache-Control headers for some queries will be:
+
+```graphql
+# Cache-Control header: no-cache, PRIVATE
+{
+  me {
+    # 5, PRIVATE
+    tasks {
+      # 50, PUBLIC
+      id # 10, PUBLIC
+      description # no-cache, PUBLIC
+    }
+  }
```

Comment:
```suggestion
  me { # 5, PRIVATE
    tasks { # 50, PUBLIC
      id # 10, PUBLIC
      description # no-cache, PUBLIC
    }
  }
```

Otherwise it is unclear which field the comment refers to